ClearKeep/Scenes/Home/Views/HomeContentView.swift

```diff
@@ -2,19 +2,87 @@
 //  HomeContentView.swift
 //  ClearKeep
 //
-//  Created by NamNH on 01/03/2022.
+//  Created by NamNH on 17/05/2022.
 //
 
 import SwiftUI
 
+private enum Constants {
+	static let cornerRadius = 16.0
+	static let hSpacing = 28.0
+	static let searchHeight = 52.0
+	static let spacing = 24.0
+}
+
 struct HomeContentView: View {
+	// MARK: - Variables
+	@Environment(\.colorScheme) var colorScheme
+	@Binding var groups: [GroupViewModel]
+	@Binding var peers: [GroupViewModel]
+	@State private(set) var isSearchView: Bool = false
+	@State private(set) var isCreateGroup: Bool = false
+	@State private(set) var isCreateMessage: Bool = false
+	@State private(set) var isNext: Bool = false
+	@State private var selectedGroup: GroupViewModel?
+	@Binding var serverName: String
+	@Binding var isExpandGroup: Bool
+	@Binding var isExpandDirectMessage: Bool
+
+	// MARK: - Body
 	var body: some View {
-		Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+		VStack(spacing: Constants.spacing) {
+			Button(action: searchAction, label: {
+				HStack(spacing: Constants.hSpacing) {
+					AppTheme.shared.imageSet.searchIcon
+						.foregroundColor(AppTheme.shared.colorSet.greyLight)
+					Text("Home.Search".localized)
+						.font(AppTheme.shared.fontSet.font(style: .input3))
+						.foregroundColor(AppTheme.shared.colorSet.greyLight)
+				}
+				.padding()
+				.frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .leading)
+			})
+			.frame(height: Constants.searchHeight)
+			.background(colorScheme == .light ? AppTheme.shared.colorSet.grey5 : AppTheme.shared.colorSet.darkgrey3)
+			.cornerRadius(Constants.cornerRadius)
+			ScrollView {
+				ListGroupView(title: "Home.GroupChat".localized + " (\(groups.count)) ",
+							  groups: groups,
+							  action: { isCreateGroup.toggle() }, onChooseGroup: { group in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Views/HomeContentView.swift#L51"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>HomeContentView.swift:51</code></td>
    </tr>
  </tbody>
</table>